.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       run: |
         # TinyXML is not installed as a workaround for https://github.com/robotology/ycm/pull/296
         C:/robotology/vcpkg/vcpkg.exe --overlay-ports=C:/robotology/robotology-vcpkg-ports --overlay-ports=${GITHUB_WORKSPACE}/custom-ports install --triplet x64-windows ace asio boost-asio boost-bind boost-process boost-dll boost-filesystem boost-system freeglut gsl eigen3 esdcan-binary glew glfw3 ode openssl libxml2 libjpeg-turbo eigen3 opencv portaudio matio sdl1 sdl2 qt5-base[latest] ipopt-binary qt5-declarative qt5-multimedia qt5-quickcontrols qt5-quickcontrols2 sqlite3[core,tool]
-
+        C:/robotology/vcpkg/vcpkg.exe list
```

Comment:
Is this like a debug print?

Reply:
Sorry, this was added intentionally to provide the details on the version to be added in the "Detailed Info" of the Release description such as in https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/tag/v0.4.0 . I updated the PR description.

Reply:
Ah, I see!